chat-client/src/client/mynahUi.ts

```diff
@@ -460,7 +461,7 @@ export const createMynahUi = (
 
         if (isPartialResult) {
             // type for MynahUI differs from ChatResult types so we ignore it
-            mynahUi.updateLastChatAnswer(tabId, { ...chatResultWithoutType, header: header })
+            mynahUi.updateLastChatAnswer(tabId, { ...chatResultWithoutType, header: header, buttons: buttons })
```

Comment:
Do we still need this if chatMessage already has header and buttons?

Reply:
yeah the types are slightly different, I pulled them out and converted them above